spock-specs/SpockTestConfig.groovy

```diff
@@ -1,5 +1,13 @@
+import java.time.Duration
+
 runner {
   parallel {
     enabled true
   }
 }
+
+timeout {
+  globalTimeout Duration.ofMinutes(1)
```

Comment:
Do you want to apply that also to the fixures?

Reply:
I don't think that is necessary in this code base.